rubocop.el

```diff
@@ -58,6 +58,11 @@
   "The command used to run RuboCop's autocorrection."
   :type 'string)
 
+(defcustom rubocop-config-file-names
+  '(".rubycop.yml" ".rubycop" "rubocop.yml" ".rubocop")
```

Comment:
rubycop? I've never seen such a config file for RuboCop. The only valid name is `.rubocop.yml`. I should now. :-)